### DISCUSSION TITLE
StabilityMatrix.db

### DISCUSSION THREAD

**Discussion by FilipeF12** [LykosAI/StabilityMatrix#922] [Q&A] [answered]
What format is the StabilityMatrix.db file? Tried a few LiteDB viewers but they don't recognize the format.

**Comment by mohnjiles** [comment]
LiteDB is correct, maybe try https://github.com/mbdavid/LiteDB.Studio ?

**Reply by FilipeF12** [reply]
I should have specified. On a Windows PC.

**Reply by mohnjiles** [reply, marked as answer]
That should work for Windows, https://github.com/mbdavid/LiteDB.Studio/releases/tag/v1.0.3

The exe is at the bottom of this page

**Reply by FilipeF12** [reply]
Perfect. Thank you for your kind help.
